base/OpenGLVisitor.cpp

```diff
@@ -14,14 +14,7 @@
 #include "Polygon.h"
 #include "BillBoard.h"
 #include "Config.h"
-
-#ifndef EM_VERTEXARRAY
-#define EM_VERTEXARRAY 0
-#endif
-
-#ifndef EM_OPENGL_LIGHTS
```

Comment:
please keep this I can be cleaned up in other change because I've seen ifdef EPENGL_LIGHTS elsewhere

Reply:
There is OPENGL_LIGHTS in AmbientLightVisitor.cpp and TextureUtil.cpp, I see EM_OPENGL_LIGHTS only in OpenGLVisitor.cpp and it doesn't seem to be used.
glEnableClientState() is used in the OpenGL ES 1.1 CM port but inside the EM_VERTEXARRAY flag I don't really understand the purpose.
As you wish, we can keep it for now.